docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -55,13 +61,15 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
       <td colspan="2" class="yes">Yes</td>
       <td class="yes">Yes</td>
       <td class="yes">Yes</td>
+      <td class="yes">Yes</td>
     </tr>
     <tr>
       <td>Ignore issues and files,<br/>configure code patterns and file extensions,<br/>manage branches</td>
       <td>No</td>
       <td colspan="2" class="yes"><a href="#change-analysis-configuration">Configurable</a></td>
       <td colspan="2" class="yes"><a href="#change-analysis-configuration">Configurable</a></td>
       <td class="yes">Yes</td>
+      <td>Inherits original permission</td><!--TODO TAROT-2414 confirm this is correct. Note on Slite: Configurable, depends on Repository permission, and only for repositories that they have access to in the git provider-->
```

Comment:
**To all reviewers**

Some of the Org Manager permissions are inherited from the original roles. Sometimes, these base permissions can be configured for the original role.

Is "Inherits original permissions" explicit enough in these instances? I left TODO notes next to each appearance of these permissions, also copying the relevant comment on the [Slite document with the permissions table](https://codacy.slite.com/app/docs/RU4Xnq0inKC-N-#38e0aec4).

Let me know if you spot inconsistencies!